stdlib/std.jsonnet

```diff
@@ -1625,7 +1625,7 @@ limitations under the License.
       [std.prune(x) for x in a if isContent($.prune(x))]
     else if std.isObject(a) then {
       [x]: $.prune(a[x])
```

Comment:
This way, every hidden field will be unhidden, and this will be a breaking change.

Also, fields referencing self/super will be frozen, and I don't think this is an useful thing after that?

Reply:
What about these other options?
1. `std.prune(a, keep_hidden=false)`
2. `std.prune(a, arrays=true, objects=true, nulls=true, hidden=true)`
3. `std.pruneEx(a, arrays=true, objects=true, nulls=true, hidden=true)`

Showing the default-state function headers here, users would then be able to override whatever they want to be pruned or not.